### ISSUE TITLE
Beautifying HTML(Angular) unindents comments

### ISSUE THREAD

**Issue by filipesilva** [Glavin001/atom-beautify#561]
Here is the gist with debug information:
https://gist.github.com/filipesilva/9d3e9a644379519e071a

Input:
```
<div>
    <div>
        <!-- comment here -->
        <div></div>
    </div>
</div>
```

Expected:
```
<div>
    <div>
        <!-- comment here -->
        <div></div>
    </div>
</div>
```

Actual
```
<div>
    <div>
<!-- comment here -->
        <div></div>
    </div>
</div>
```

If I remember correctly, the HTML (Angular) beautifying is handled by Prettydiff EJS beautifying. I tried this on Prettydiff's website and the result was the expected one, whereas on Atom I got the actual one.

Earlier today I reported an issue and you recommended me to reinstall `atom-beautify` which I have, so I don't think it's a version thing.

**Comment by Glavin001** [comment]
@prettydiff any ideas?

**Comment by filipesilva** [comment]
It's worth mentioning that multiline comments retain the proper indentation on subsequent lines

Input:
```
<div>
  <div>
    <!-- comment here 
    much multi
    such line
    wow -->
    <div></div>
  </div>
</div>
```

Expected:
```
<div>
  <div>
    <!-- comment here 
    much multi
    such line
    wow -->
    <div></div>
  </div>
</div>
```

Result:
```
<div>
  <div>
<!-- comment here 
    much multi
    such line
    wow -->
    <div></div>
  </div>
</div>
```

**Comment by prettydiff** [comment]
I cannot reproduce this error.  Pretty Diff does have an option to remove indentation prior to a comment: [comments](http://prettydiff.com/documentation.xhtml#comments) with value `noindent`

**Comment by filipesilva** [comment]
Ah that might be it, yes! Going over the debug info I can see `comments=noindent`

```
2015-09-22T23:06:58.654Z - verbose: [C:\Users\filipe.silva\.atom\packages\atom-beautify\src\beautifiers\beautifier.coffee] prettydiff inchar= , insize=2, objsort=false, preserve=all, cssinsertlines=undefined, comments=noindent, force=undefined, quoteConvert=undefined, vertical=none, wrap=80, space=false, noleadzero=undefined, endcomma=false, source=<div>
```

Must be a default somewhere?

**Comment by Glavin001** [comment]
OH.

This makes sense. It appears to fallback to `noindent`: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L20 

A simple Pull Request could fix this :wink:.
Thanks!

**Comment by filipesilva** [comment]
Yeah I had just found that exact same place :D 

I'll make a PR.

**Comment by Glavin001** [comment]
Published patch to v0.28.14
